abuild.in

```diff
@@ -2424,13 +2424,13 @@ deps() {
 	if [ -n "$CBUILDROOT" ]; then
 		[ -z "$_quiet" ] && msg "Installing for host:$hostdeps"
 		$SUDO_APK add $_quiet --root "$CBUILDROOT" --arch "$CTARGET_ARCH" --repository "$REPODEST/$repo" $apk_opt_wait \
-			--no-scripts --virtual .makedepends-$pkgname $hostdeps || return 1
+			--no-scripts --virtual .hostdepends-$pkgname $hostdeps || return 1
 	fi
 }
 
 undeps() {
 	local _quiet="$@"
-	$SUDO_APK del $_quiet $apk_opt_wait .makedepends-$pkgname || :
+	$SUDO_APK del $_quiet $apk_opt_wait .makedepends-$pkgname .hostdepends-$pkgname || :
```

Comment:
Should'nt the `.hostdepends-$pkgname` be removed within the `if [ -n "$CBUILDROOT" ]; then` and shouldn't it have a `--root "$CBUILDROOT"` option too?